pkg/workloadmeta/collectors/internal/containerd/image_sbom_trivy_test.go

```diff
@@ -0,0 +1,142 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+//go:build containerd && trivy && test
+
+package containerd
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/CycloneDX/cyclonedx-go"
+	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
+	trivydx "github.com/aquasecurity/trivy/pkg/sbom/cyclonedx"
+)
+
+func Test_updateSBOMMetadata(t *testing.T) {
+	type args struct {
+		sbom        *workloadmeta.SBOM
+		repoTags    []string
+		repoDigests []string
+	}
+	tests := []struct {
```

Comment:
I think that for completeness it would be good to add a test that updates the SBOM to remove a tag and repo digest. I think that's something that should be supported, correct me if I'm wrong.

Reply:
Nice catch. I wasn't sure if it was a good idea because RepoDigests and RepoTags are correlated with the image's name in the remote repository. I think if they are removed, it's only locally. So we lose information.
In any case, I think the local collection should stay consistent with the local information. Correlation with the remote registry should be something for the back-end.